python/linked_list.py

```diff
@@ -1,19 +1,68 @@
-class LinkList:
-  # write your __init__ method here that should store a 'head' value which the first Node in the LinkedList and a 'length' value which is the total number of Nodes in the LinkedList
+class LinkList():
+  def __init__(self, head=None):
+    self.head = head
+    self.length = 1
```

Comment:
Since head defaults to none, it's possible to start with a length of 0, so something with the ternary operator like:
```suggestion
    self.length = 0 if head == None else 1
```
may work